### PR TITLE
Xamarin documentation

### PR DESCRIPTION
**Pull request by okdistribute**
- [x] Installation pointer to tutorial
- [x] Remove 'coming soon xamarin' from languages grid
- [x] Add tasks tutorial
- [x] Add tasks sample code to github.com/getditto/samples
- [x] Test nuget installation to make sure no extra glue code or configuration is needed

### CODE REVIEW

**Comment by phatblat** [comment]
Currently, Xamarin.iOS apps need to provide a working directory inside the app's sandbox:

```csharp
NSFileManager fileManager = new NSFileManager();
NSUrl url = fileManager.GetUrl(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomain.User, null, true, out NSError error);
if (error != null)
{
    Console.WriteLine($"Error creating Documents directory: {error.LocalizedDescription}");
}
url = url.Append("ditto", true);

fileManager.CreateDirectory(url, true, null, out error);
if (error != null)
{
    Console.WriteLine($"Error creating ditto directory: {error.LocalizedDescription}");
}

string workingDir = url.Path;

DittoIdentity identity = DittoIdentity.OnlinePlayground(appID: Constants.OnlineAppId, token: Constants.OnlinePlaygroundToken, workingDir: workingDir);
var ditto = new Ditto(identity, workingDir);
```

Without this, the default directory used by the SDK won't be writable and an exception will be thrown.

**Comment by phatblat** [comment]
I thought I had created an issue to alleviate the need to always provide a custom working directory but I guess not. Created getditto/ditto#6068

File: src/components/Languages.js [hunk 1]
```diff
@@ -52,12 +51,6 @@ export default [
     name: 'C++',
     icon: CPlusPlus 
   },
-  {
```
Comment (okdistribute) [inline src/components/Languages.js:55]: Do you think we should leave this in and send people to C#?

**Comment by okdistribute** [comment]
Can merge this once 2.0.1 is out, this PR fixes a crucial bug https://github.com/getditto/ditto/pull/6082

**Comment by okdistribute** [comment]
2.0.1-expirimental is out with xamarin support, we're giving it to DextraData on monday

**Pull request merged**